compiler/rustc_error_messages/locales/en-US/session.ftl

```diff
@@ -0,0 +1,13 @@
+incorrect_cgu_reuse_type =
+    CGU-reuse for `{$cgu_user_name}` is `{$actual_reuse}` but should be `{$at_least}``${expected_reuse}`
+
+cgu_not_recorded =
+    CGU-reuse for `{$cgu_user_name}` is (mangled: `{$cgu_name}`) was not recorded`
+
+feature_gate_error = {$explain}
+
+feature_diagnostic_for_issue =
+    see issue #{$n} <https://github.com/rust-lang/rust/issues/{$n}> for more information
+
+feature_diagnostic_help =
+    add `#![feature({$feature})]` to the crate attributes to enable
```

Comment:
These diagnostics slugs are missing their `session_` prefix - all the Fluent files are part of one global namespace, so if two files were to both declare e.g. `feature_gate_error`, it would lead to a collision. That's why slugs start with the crate name by convention, which is then chopped off by the `fluent_messages!{}` macro (#100675 will ensure that this is caught by CI).

Reply:
Hi @Xiretza 

Thanks for your review :)

I've just changed it as requested and I've amended my commit.

Please, see the diff [here](https://github.com/rust-lang/rust/commit/a446e0e6c828a376d90a29ae079334c7d3933e72#diff-538f2311cb0739204cc09203e4647eb4a83ccf88c8aad655d5b40245d0a43290).